cdec/installation-manager-resources/src/main/resources/im-scripts/install-codenvy.sh

```diff
@@ -928,6 +941,15 @@ doCheckAvailableResourcesLocally() {
         exit 1;
     fi
 
+    if [[ ${writeAccessIssueFound} == true ]]; then
+        if [[ -n $INSTALL_DIRECTORY ]]; then
+            println "$(printError "ERROR: Installation directory") $(printImportantLink $INSTALL_DIRECTORY) $(printError "cannot be written to.")"
```

Comment:
That is how it defined [here](https://jira.codenvycorp.com/browse/CODENVY-235?focusedCommentId=273935&page=com.atlassian.jira.plugin.system.issuetabpanels:comment-tabpanel#comment-273935).